doc/source/contributing/sphinx_doc_build.md

```diff
@@ -1,15 +1,133 @@
 (sphinx_doc_build)=
 # Building the documentation
+Want to help with ArviZ documentation? There are several ways you could assist. You could help by suggesting changes via a pull request (PR), improving our documentation, reporting, and fixing issues. ArviZ documentation is constantly being improved and expanded by users and contributors, if you see anything that’s missing or have ideas on how to improve the existing documentation, please consider getting involved.
 
-The preferred workflow for contributing to ArviZ is to fork
-the [GitHub repository](https://github.com/arviz-devs/arviz/),
-clone it to your local machine, and develop on a feature branch. For a detailed description of the recommended development process, see the step-by-step guide {ref}`here <pr_steps>`.
+We recommend getting familiar with our [GitHub repository](https://github.com/arviz-devs/arviz/) before contributing to the ArviZ documentation.
+Our documentation is built using Sphinx and written in Markdown, but can also be written using reStructuredText or Jupyter notebook. The files are appended with the extensions .md, .rst, and .ipynb, respectively.
 
-If you are using Docker, see {ref}`building_doc_with_docker`.
+## Table of content:
+- [Getting started with documentation](Getting_started_with_documentation)
+- [Using Git on the command-line](Using_Git_on_the_command-line)
+- [Editing documents using Git](Editing_documents_using_Git)
+- [Building documents with Docker](Building_documents_with_Docker)
+- [Pull request checks](Pull_request_checks)
+- [Previewing doc changes](Previewing_doc_changes)
+- [Resources](Resources)
+   - Markdown guide
+   - reStructuredText guide
+   
+
+## Getting started with documentation
+
+Creating or improving the documentation requires that you build it locally. There are a few methods to choose from:
+
+- Using Git on the command line
+- Building documentation with Docker
+
+## Using Git on the command-line
+
+The preferred workflow for contributing to ArviZ documentation is to fork the [GitHub repository](https://github.com/arviz-devs/arviz/), clone it to your local machine, develop on a feature branch, commit the changes, push it to GitHub and open a new pull request (PR). See the step-by-step guide {ref}`here <pr_steps>`.
+
+You'll need to install and configure Git and have a Github account to work with the document locally. Alternatively, you could work with the documents using Docker. See building document with Docker {ref}`building_doc_with_docker`.
+
+## Editing documents using Git
```

Comment:
I am a bit worried about the duplication of content between this section and https://python.arviz.org/en/latest/contributing/pr_tutorial.html. I think ideally the PR step by step guide would be used and linked from both instructions to contribute code and instructions to contribute docs. We could improve the PR step by step guide and make sure it is not code specific to try avoiding duplicating all the fork+clone+branch instructions as they are common to any code or doc contribution. What do you think?

Reply:
It appears the PR step-by-step is geared towards developers who want to contribute code. I agree, we could improve that to prevent duplication. In that case, I'll remove that section.